restonomer-docs/source/restonomer_transformations.md

```diff
@@ -282,3 +282,56 @@ The transformed response will now have the columns with the desired data types:
   "col_B": "[{'ZipCodeType':'STANDARD','Zipcode':704}]"
 }
 ```
+## ReplaceStringInColumnValue
+
+It lets the user cast the data type of multiple columns to the desired different types at once.
```

Comment:
This description is wrong. Its copied from CastColumns. Please correct it.

Reply:
Done